lib/simple-cookie-manager.js

```diff
@@ -4,24 +4,50 @@
  * Released under the GNU GENERAL PUBLIC LICENSE
 */
 
-var SimpleCookieManager = function(){
-  this.readCookie = function(name){
- var _search = name + "=";
- if (document.cookie.length > 0) {  
-  _cookiePos = document.cookie.indexOf(_search);
-  if (_cookiePos != -1) { 
-   _cookiePos += _search.length;  
-   _finish = document.cookie.indexOf(";", _cookiePos); 
-   if (_finish == -1) 
-     _finish = document.cookie.length;
-   return unescape(document.cookie.substring(_cookiePos, _finish));
-  }
- }
-}
- this.writeCookie = function(name, val, h){
-  var _today = new Date();
-  var _cookend = new Date();
-  _cookend.setTime(_today.getTime() + 1000*60*60*h);
-  document.cookie = name + "=" + escape(val) + "; expires=" + _cookend.toGMTString();
-}
-}
\ No newline at end of file
+var SimpleCookieManager = function(){};

```

Comment:
I would also suggest the more modern `Object.create` syntax
<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create> if backwards compatibility is not a big deal;

It is also possible to use ES6/ES7 and transpile it using something like Babel
( <https://babeljs.io> ) which will make the code leaner.

I didn’t go that way because for the sake of this module plain old javascript is good enough; and Babel might be an overkill.

An additional suggestion would be to use namespaces instead of global functions to reduce naming conflicts, which still might be an overkill.

And instead of naming the object as `SimpleCookieManager` maybe a “simpler” name would stick more like…

`SimpleCookie.read(blah)` 
`SimpleCookie.wr9te(blah, blah)`

(no offense, and) I fucking hate objects that end with `er`; especially Managers, Controllers, Renderers, et. al.; more often than not, they are precursors of a smelly code.